js/deer-record.js

```diff
@@ -519,7 +519,7 @@ async function create(obj, attribution, evidence) {
             continue
         }
         let annotation = {
-            "@context": "",
+            "@context": mint["@context"],
```

Comment:
Who is mint? I'll watch this space.

Reply:
Yeah, have a look.  Mint the Annotation generated from the objects fed into this function.  It contains the context (or empty string) supplied in that object.

Reply:
Since each Annotation does not have its own context (like motivation), I assume the context of the Annotation should be same as the context of the parent entity.

Reply:
You will know why this will work for Lived Religion if you see the context.json changes in the pull request https://github.com/CenterForDigitalHumanities/deer-LR/pull/158/.  The lived religion context handles Web Annotation LD for Annotations, which is in all deer-context attributes in forms of Lived Religion.  So the only place I need to handle an array of context in Lived Religion is on `place.html` where we generate GeoJSON Web Annotations.  The rest are just the single Lived Religion context URL.

Reply:
This is in unused code.